js/model.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+function ProductModel(name, price)
+{
+  this.name = name;
+  this.price = price;
+}
+//goal is to store description in a variable of its own.
+function ProductDescription(description)
+{
+  this.description = description;
+}
+
+function ProductImage(image)
```

Comment:
I would combine this into `ProductDescription`. No need for separate arrays; just make one array of all your products.